site/app/templates/grading/UploadImagesForm.twig

```diff
@@ -3,8 +3,9 @@
 {% block title %}Upload Student Photos{% endblock %}
 {% block body %}
     <p>
-        Follow the instructions to download images of students on:<br>
-        <a href="http://submitty.org/instructor/student_photos">Student Photos</a><br><br>
+        Instructions to download images of students: 
+        <a target=_blank href="http://submitty.org/instructor/student_photos"> <i class="far fa-images" style="color:Dodgerblue"></i></a>
```

Comment:
Should this also be a question mark, I think thats more helpful because its consistent with everything else. The phot image makes me think that it does something other than bring to a help page.

Reply:
Probably, I just saw that and thought it looked like an interesting option.